### DISCUSSION TITLE
Sales>Orders - Find and Export CSV - how to add/change fields in this report?

### DISCUSSION THREAD

**Discussion by gnossosp** [openstore-ecommerce/OpenStore#299] [Q&A]
The Sales>Orders "Find an Order" search feature containing From and To dates as well as the Status drop down  will export orders to a CSV format.  Fields include: Order number, First, Last, Email PO Box (line 2) City, Zip, Country, Order Date as well as price, shipping and VAT fields.  I would like to tweak to add street address 1 as well as phone number records and ship to address fields. I would like to remove the Country field since all these orders will be in one Country.

Unfortunately I am not sure how to accomplish this.  Could someone please point me in right direction?

Thanks

**Comment by DNNMonster** [comment]
I think you might want to investigate OS_Reports.  If it will work for you it will simplify things by letting you create the SQL to your liking. https://github.com/openstore-ecommerce/OS_Reports
